operator/pkg/reconciliation/context.go

```diff
@@ -37,6 +37,8 @@ type ReconciliationContext struct {
 	Services               []*corev1.Service
 	desiredRackInformation []*RackInformation
 	statefulSets           []*appsv1.StatefulSet
+	dcPods                 []*corev1.Pod
```

Comment:
Can we have comments here to describe exactly what each field is?  Especially when we have two fields for something, like pods.

Reply:
This is actually a discussion point, because we (Jim and I) talked about it and weren't sure why we were differentiating between cluster pods and dc pods. Almost everything goes by dc pods, but 2 or 3 functions specifically used cluster pods. Ideally we can just use dcPods for everything if we don't have a good reason to use cluster pods.

Reply:
yeah I don't think we can get rid of clusterPods, sadly